README.md

```diff
@@ -6,7 +6,7 @@
 
 [![Gitter](https://badges.gitter.im/Join%20Chat.svg)](https://gitter.im/diydrones/dronekit-python?utm_source=badge&utm_medium=badge&utm_campaign=pr-badge&utm_content=badge)
 
-This package contains the python language bindings for the drone API and the associated MAVProxy module.
+This package contains the python language bindings for the drone API.
```

Comment:
This isn't relevant anymore, what @geeksville was referring to as "drone API" is DK-Cloud and the "MAVProxy module" is DK-Python, I nominate @hamishwillee to write a better description to what DK-Python is :sunglasses: